index.js

```diff
@@ -342,5 +92,12 @@ module.exports = {
 
     // self.apos.define('apostrophe-cursor', require('./lib/cursor.js'));
 
+    if (options.experiencesQuery) {
+      getExperiences(options)
+        .then(experiences => {
+          console.log('EXPERIENCES: ', experiences);
+          self.experiences = experiences;
```

Comment:
This can't just "hang out" and happen asynchronously some indeterminate time after the module has already reported it is ready.

It should happen in a structured way. Either a method that invokes a callback properly on success or error, invoked by `afterConstruct`, or perhaps a `modulesReady` promise event handler.